ui/pages/token-details/token-details-page.js

```diff
@@ -46,8 +45,11 @@ export default function TokenDetailsPage() {
     isEqualCaseInsensitive(address, tokenAddress),
   );
 
-  const { tokensWithBalances } = useTokenTracker({ tokens: [token] });
+  const { tokensWithBalances } = useTokenTracker({
+    tokens: token ? [token] : [],
```

Comment:
When the user did not import the token; the `token `variable will be `undefined `resulting in calling the `useTokenTracker `
with` [undefined]` instead of an empty array [] which will lead to an error when creating the `TokenTracker `instance.

Reply:
Hey @sahar-fehri neat fix ! do you mind adding the explanation of `token` undefined in the code so others would know the reason of this conditional render. wdyt?

Reply:
Definitely will add the comment! 🙏